frontend/src/app/admin/superadmin/superadmin.component.scss

```
@@ -90,14 +90,40 @@
   }
 
   .primary-action {
-    background: var(--accent-600);
+    background: linear-gradient(135deg, #2563eb 0%, #1d4ed8 100%);
     color: #fff;
+    border: 1px solid #1d4ed8;
+    box-shadow: 0 10px 24px rgba(37, 99, 235, 0.25);
```

Stylelint (17.2.0)

[error] Expected "0.25" to be "25%" (alpha-value-notation)

(alpha-value-notation)